src/components/Card/Card.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactElement, useState } from "react";
+import { StyleSheet, View, Text } from "react-native";
+import { colors } from "../../../assets/theme/colors";
+import { NavigationScreenProp } from "react-navigation";
+import { SvgProps } from "react-native-svg";
+
+export const Card: React.FC<{
+  label: string,
+  svgIcons: Array<(svgProps?: SvgProps)=>ReactElement>,
+  navigationScreen: string,
+  navigation: NavigationScreenProp<any,any>    // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
please explain - why there is use of any?

Reply:
YEs, For navigation we have it as any for now, will remove it soon. It's. a part of refactoring. But for now can you please resolve the coment.